feature/amazon/src/test/java/com/revenuecat/purchases/amazon/handler/UserDataHandlerTest.kt

```diff
@@ -54,8 +56,10 @@ class UserDataHandlerTest {
     fun setup() {
         purchasingServiceProvider = PurchasingServiceProviderForTest()
         mainHandlerCallbacks.clear()
+        timestampUtils = mockk()
```

Comment:
I would typically make `TimestampUtils` an interface (and call it `TimestampProvider`) and have a `DefaultTimestampProvider`. Then create a new class for tests that returns whatever. This is to avoid mocks. But it's a personal choice, so if you prefer this go for it. Just wanted to mention an alternative :)

Reply:
Yeah I think both options work. That said, since we already have a `DateProvider`, I think it's better to keep it consisten, so I will follow the same pattern.